src/condor_schedd.V6/schedd.cpp

```diff
@@ -218,7 +218,9 @@ void AuditLogNewConnection( int cmd, Sock &sock, bool failure )
 		return;
 	}
 
-	if ( !strcmp( get_condor_username(), sock.getOwner() ) ||
+	std::string uid_domain;
```

Comment:
don't param here.  schedd should have this information stored

Reply:
Good catch!  Indeed, it's `qmgmt.cpp` that doesn't have this readily available.